src/panels/DetailPanel.tsx

```diff
@@ -89,31 +71,22 @@ export default class DetailPanel extends React.Component<Props, State> {
     }
 
     render() {
-        if (!this.state.hidden) {
-            if (this.state.type === "elem") {
-                return (<DetailElement projectLanguage={this.props.projectLanguage}
-                                       save={this.save} ref={this.detailElem}
-                                       handleWidth={this.props.handleWidth}
-                                       performTransaction={this.props.performTransaction}
-                                       error={this.props.error}
-                                       id={this.props.id}
-                                       updateDetailPanel={this.props.updateDetailPanel}
-                />);
-            } else if (this.state.type === "link") {
-                return (<DetailLink
-                    error={this.props.error}
-                    handleWidth={this.props.handleWidth}
-                    projectLanguage={this.props.projectLanguage}
-                    performTransaction={this.props.performTransaction}
-                    save={this.save} ref={this.detailLink}
-                    id={this.props.id}
-                    updateDetailPanel={this.props.updateDetailPanel}
-                />);
-            }
-        } else {
-            return (<div/>);
-        }
-
+        return (<div>
+            <DetailElement projectLanguage={this.props.projectLanguage}
+                           save={this.save} ref={this.detailElem}
+                           handleWidth={this.props.handleWidth}
+                           performTransaction={this.props.performTransaction}
+                           error={this.props.error}
+                           updateDetailPanel={this.props.updateDetailPanel}
+            />
+            <DetailLink
```

Comment:
You should probably choose one code style and just go with that :-) Both components here (DetailElement and DetailLink) are formatted differently.